app/src/main/kotlin/com/wabadaba/dziennik/api/notification/NotificationHelper.kt

```diff
@@ -0,0 +1,35 @@
+package com.wabadaba.dziennik.api.notification
+
+import android.app.Notification
+import android.app.NotificationManager
+import android.app.NotificationChannel
+import android.os.Build
+import android.support.annotation.RequiresApi
+
+
+class NotificationHelper(private val notificationManager: NotificationManager) {
+    companion object {
+        val NOTIFICATION_CHANNEL_ID = "com.wabadaba.dziennik.notifications"
+        val NOTIFICATION_TAG = "com.wabadaba.dziennik"
```

Comment:
shouldn't this be read from some kind of android service?

Reply:
These are notification channels id / tag. You need to set them manually